umh.docs.umh.app/content/en/docs/getstarted/dataacquisitionmanipulation.md

```diff
@@ -25,10 +25,19 @@ The UMH includes 3 pre-configured data simulators for testing connections:
 OPC UA, often complex, can be streamlined using our Benthos-based OPC UA connector
 accessible from the Management Console.
 
+### Visualize Connections or Network Devices
+
+From the **Network View**, you can visualize the connections or network devices a UMH
+instance is connected to. This view provides the latest status of both the instances and
+the network devices, the latter also showing the latency in milliseconds.
+
+![Network View](/images/getstarted/dataAcquisitionManipulation/networkViewDevice.png?width=80%)
+
 ### Create a Connection with the Management Console
 
-After logging into the Management Console and selecting your instance, navigate
-to the **Connection Management** tab, where you'll find all your connections alongside their status.
+From the **Component View**, in the **Connection Management** tab, you'll also find all
```

Comment:
Try to keep the wording similar. "In order to add a new netowrk device or external service, go to COmponent view into the Connection Management tab ...."